python_sdk/infrahub_sdk/client.py

```diff
@@ -1492,7 +1851,7 @@ def get_diff_summary(
 
     def allocate_next_ip_address(
         self,
-        resource_pool: InfrahubNodeSync,
+        resource_pool: CoreNodeSync,
```

Comment:
Looks like we're missing overloads here?

Reply:
Good catch I fixed it